test/files/qty000/pass-qty000-not_activated_no_relating_element.ifc

```diff
@@ -0,0 +1,31 @@
+ISO-10303-21;
+HEADER;
+FILE_DESCRIPTION(('ViewDefinition [CoordinationView]'),'2;1');
+FILE_NAME('','2024-07-28T20:26:00',(''),(''),'IfcOpenShell-0.7.11-d51fa2c5f','IfcOpenShell-0.7.11-d51fa2c5f','');
+FILE_SCHEMA(('IFC4X3_ADD2'));
+ENDSEC;
+DATA;
+#1=IFCPERSON($,$,'',$,$,$,$,$);
+#2=IFCORGANIZATION($,'',$,$,$);
+#3=IFCPERSONANDORGANIZATION(#1,#2,$);
+#4=IFCAPPLICATION(#2,'0.7.11-d51fa2c5f','IfcOpenShell-0.7.11-d51fa2c5f','');
+#5=IFCOWNERHISTORY(#3,#4,$,.NOTDEFINED.,$,#3,#4,1722198360);
+#6=IFCDIRECTION((1.,0.,0.));
+#7=IFCDIRECTION((0.,0.,1.));
+#8=IFCCARTESIANPOINT((0.,0.,0.));
+#9=IFCAXIS2PLACEMENT3D(#8,#7,#6);
+#10=IFCDIRECTION((0.,1.));
+#11=IFCGEOMETRICREPRESENTATIONCONTEXT($,'Model',3,1.E-05,#9,#10);
+#12=IFCDIMENSIONALEXPONENTS(0,0,0,0,0,0,0);
+#13=IFCSIUNIT(*,.LENGTHUNIT.,$,.METRE.);
+#14=IFCSIUNIT(*,.AREAUNIT.,$,.SQUARE_METRE.);
+#15=IFCSIUNIT(*,.VOLUMEUNIT.,$,.CUBIC_METRE.);
+#16=IFCSIUNIT(*,.PLANEANGLEUNIT.,$,.RADIAN.);
+#17=IFCMEASUREWITHUNIT(IFCPLANEANGLEMEASURE(0.017453292519943295),#16);
+#18=IFCCONVERSIONBASEDUNIT(#12,.PLANEANGLEUNIT.,'DEGREE',#17);
+#19=IFCUNITASSIGNMENT((#13,#14,#15,#18));
+#20=IFCPROJECT('1Dw0Rp2e9CJ8l4rbpvS0W2',#5,'',$,$,$,$,(#11),#19);
+#21=IFCELEMENTQUANTITY($,$,$,$,$,$);
```

Comment:
This doesn't pass schema validation. For the entities that the scenario is about I think we should put in a little bit extra effort to produce a valid file.

Reply:
See reply at https://github.com/buildingSMART/ifc-gherkin-rules/pull/247/files#r1705935701

Reply:
I've modified the testfiles so that they all pass the schema and syntax check
https://github.com/buildingSMART/ifc-gherkin-rules/commit/b695316aff4353c2959f8e21092f5b2c0bd69ecb